### ISSUE TITLE
Icon is not required in Drawer Section Item API

### ISSUE THREAD

**Issue by Falkyouall** [xotahal/react-native-material-ui#238]
Your api states that Value is required as Drawer Section Item

`items: PropTypes.arrayOf(PropTypes.shape({
        icon: PropTypes.string,
        value: PropTypes.oneOfType([PropTypes.string, PropTypes.element]).isRequired,
        label: PropTypes.string,
        onPress: PropTypes.func,
        onLongPress: PropTypes.func,
        active: PropTypes.bool,
        disabled: PropTypes.bool,
    }))`

your implementing it like

`<ListItem
      dense
      key={item.icon}
      leftElement={item.icon}
      centerElement={item.value}
      onPress={item.onPress}
     style={style}
     />`

but it gets not used as Key, instead icon is used as key... okay then let it be, but please add to your API that icon is required! thank you.

**Comment by aburai** [comment]
@Falkyouall i don't think so, it is conceivable that someone use identical icons for the list items and we get => Encountered two children with the same key, `...`. Keys should be unique so that components ...
the same for value and there is currently an unused item prop label.

we could simply use the index of the map function in this case. Any other ideas?
